### PR TITLE
Update documentation in Japanese

### PR DESCRIPTION
**Pull request by KengoTODA**
This PR adds translation for #124. 
It also updates existing .po files to follow recent updates in English one.

@wreulicke or @orihalcon128 ,
Could you review this Japanese translation? Thanks in advance!

[//]: # (rtdbot-start)

URL of RTD documents:
en: https://spotbugs.readthedocs.io/en/update-japanese-doc/
ja: https://spotbugs.readthedocs.io/ja/update-japanese-doc/

[//]: # (rtdbot-end)

### CODE REVIEW

File: docs/locale/ja/LC_MESSAGES/annotations.po [hunk 1]
```diff
@@ -0,0 +1,246 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016-2018, spotbugs community
```
Comment (ThrawnCA) [inline docs/locale/ja/LC_MESSAGES/annotations.po]: 2019?

File: docs/locale/ja/LC_MESSAGES/annotations.po [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+# This file is distributed under the same license as the spotbugs package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
```
Comment (ThrawnCA) [inline docs/locale/ja/LC_MESSAGES/annotations.po]: Should probably populate this.

File: docs/locale/ja/LC_MESSAGES/annotations.po [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: spotbugs 4.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-11-05 12:44+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
```
Comment (ThrawnCA) [inline docs/locale/ja/LC_MESSAGES/annotations.po]: Should probably populate or drop this.

File: docs/locale/ja/LC_MESSAGES/annotations.po [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```
Comment (ThrawnCA) [inline docs/locale/ja/LC_MESSAGES/annotations.po]: Please populate this.

File: docs/locale/ja/LC_MESSAGES/annotations.po [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+"Language-Team: LANGUAGE <LL@li.org>\n"
```
Comment (ThrawnCA) [inline docs/locale/ja/LC_MESSAGES/annotations.po]: Please populate this.

File: docs/locale/ja/LC_MESSAGES/annotations.po [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+#: ../../annotations.rst:94
+msgid "edu.umd.cs.findbugs.annotations.UnknownNullness"
+msgstr "edu.umd.cs.findbugs.annotations.UnknownNullness"
+
+#: ../../annotations.rst:95
+msgid ""
+"Used to indicate that the nullness of element is unknown, or may vary in "
+"unknown ways in subclasses."
+msgstr ""
+"修飾された要素がnullになるかどうかが不明な際、サブクラスによって決定される歳に利用します。"
```
Comment (orihalcon128) [inline docs/locale/ja/LC_MESSAGES/annotations.po]: typo
歳->際

```suggestion
"修飾された要素がnullになるかどうかが不明な際、またはサブクラスによって決定される際に利用します。"
```

File: docs/locale/ja/LC_MESSAGES/annotations.po (hunk 2 above)
Comment (KengoTODA) [reply docs/locale/ja/LC_MESSAGES/annotations.po]: It seems that the email address is not configurable. Let's put it aside for now.
https://shimizukawa-sphinx.readthedocs.io/en/latest/_modules/sphinx/builders/gettext.html

File: docs/locale/ja/LC_MESSAGES/annotations.po (hunk 3 above)
Comment (KengoTODA) [reply docs/locale/ja/LC_MESSAGES/annotations.po]: It seems that the email address is not configurable. Let's put it aside for now.
https://sphinx.readthedocs.io/en/latest/_modules/sphinx/builders/gettext.html

File: docs/locale/ja/LC_MESSAGES/annotations.po (hunk 4 above)
Comment (KengoTODA) [reply docs/locale/ja/LC_MESSAGES/annotations.po]: It seems that the email address is not configurable. Let's put it aside for now.
https://sphinx.readthedocs.io/en/latest/_modules/sphinx/builders/gettext.html

File: docs/locale/ja/LC_MESSAGES/annotations.po (hunk 5 above)
Comment (KengoTODA) [reply docs/locale/ja/LC_MESSAGES/annotations.po]: It seems that the email address is not configurable. Let's put it aside for now.
https://sphinx.readthedocs.io/en/latest/_modules/sphinx/builders/gettext.html

File: docs/locale/ja/LC_MESSAGES/annotations.po (hunk 1 above)
Comment (KengoTODA) [reply docs/locale/ja/LC_MESSAGES/annotations.po]: solved by ea8558eeb35fafdd894c90bdf6c05f113c0199f9

**Comment by KengoTODA** [comment]
@orihalcon128 @ThrawnCA Updated, please check. Thanks for your review!

**Pull request merged**